### PR TITLE
Gh 445/open tx

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Added opening of the transaction list screen
- Added opening of the details screen when we have transaction hash
- Created placeholders for registering/unregistering

### CODE REVIEW

File: Multisig/Logic/Models/Safe.swift [hunk 1]
```diff
@@ -78,15 +78,19 @@ extension Safe: Identifiable {
...
         let fr = Safe.fetchRequest().by(address: address)
-        
         guard let safe = try? context.fetch(fr).first else { return }
-        
         safe.name = name
-
         App.shared.coreDataStack.saveContext()
     }
+
+    static func select(address: String) {
```
Comment (MouazAlzahabi) [inline Multisig/Logic/Models/Safe.swift:87]: Do we need this?, we used to have Selection.select right?

Comment (DmitryBespalov) [reply Multisig/Logic/Models/Safe.swift:87]: no, there's no Selection.select(address), so I've added this static method when I don't have the Safe object

File: Multisig/UI/App/RemoteNotificationHandler.swift [hunk 2]
```diff
@@ -156,6 +180,27 @@ class RemoteNotificationHandler {
         }
     }
 
+    // MARK: - Registering in the service
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/RemoteNotificationHandler.swift:178]: In my PR I add the implementation for all those functions, after the merge you can remove them

File: Multisig/UI/App/ViewState.swift [hunk 3]
```diff
@@ -20,6 +20,9 @@ class ViewState: ObservableObject {
         state = to
     }
 
+    @Published
+    var presentedSafeTxHash: String?
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/ViewState.swift:24]: All good, but I think we need to wrap it in some structure

File: Multisig/UI/App/RemoteNotificationHandler.swift (hunk 2 above)
Comment (DmitryBespalov) [reply Multisig/UI/App/RemoteNotificationHandler.swift:178]: ok

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/App/ViewState.swift (hunk 3 above)
Comment (DmitryBespalov) [reply Multisig/UI/App/ViewState.swift:24]: we'll do it in another pass over the whole notifications code next week

**Pull request merged**